.github/workflows/config/spelling_allowlist.txt

```diff
@@ -91,9 +95,11 @@ entanglers
 enum
 et
 executables
+https
 inlining
 instantiation
 instantiations
+json
```

Comment:
Please captitalize in text (JSON) - lowercase spelling shouldn't be listed here.